crimeApi/route/image.js

```diff
@@ -0,0 +1,46 @@
+//import express
+var express = require('express');
+
+//We need to import the route module
+var router = express.Router();
+
+/*We need to import Multer 
```

Comment:
I think you can delete "We need to import Multer" and "please install the multer package". Lines 9 & 10 are what we actually need to know.